app/views/articles/show.html.slim

```diff
@@ -1,8 +1,16 @@
 h1 #{article.title}
-p
-  | #{article.content}
+p #{article.content}
 
 - if allowed_to?(:create?, article, with: Users::Articles::PublicationPolicy)
   = button_to "Publish", users_article_publications_path(article), class: "btn btn-primary"
 - if allowed_to?(:destroy?, article, with: Users::Articles::PublicationPolicy)
   = button_to "Back to Draft", users_article_publication_path(article), method: :delete, class: "btn btn-secondary"
+
+- if (current_user.present? && article.published?) || comments.present?
```

Comment:
What do you think if we always show `Comments` section. In this case we can escape the duplicate conditions.
We can add additional condition to `app/views/comments/_list.html.slim` partial and show "No comments" text if where are no comments. In this case, we can always show comments section and we can always render `comments/_list` partial.